flybywiresim/aircraft
# fix(press): increase margin for residual pressure ECAM warning

Fixes #6071

## Summary of Changes

Bug fix, introduced with #5504. After landing (FWC Flight Phase 10), when shutting down engines with APU bleed on, the residual pressure in the cabin is higher than the threshold for the "EXCES RESIDUAL PR" ECAM warning. This is because the packs stay on, creating a tiny amount of differential pressure. 

This pull request increases the threshold for the EXCES RESIDUAL PR warning to appear, from 0.01 psi to 0.03 psi.

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/60938089/138589727-9be0a60f-5c62-4c33-90b3-90c65bcccb17.png)


## References

Although not directly stated, the document: _Residual Cabin Pressure, MICHEL PALOMEQUE, Safety first no3 December 2006_, mentions a margin of 2.5hPa before the residual pressure warning on doors triggers (this is between 0.03 and 0.04 psi). I haven't been able to find any reference directly related to the ECAM warning.




## Additional context



Discord username (if different from GitHub): Commander Ga#1548

## Testing instructions

1. Spawn at the gate or stand with engines off
2. Turn on the APU and APU bleed
3. Verify no ECAM warning
4. Change the Simvar L:A32NX_FWC_FLIGHT_PHASE to number 10
5. Verify no ECAM warning
6. Manually close the outflow valve
7. Verify EXCES RESIDUAL PR warning comes up as the cabin pressurizes.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
